slang.h

```diff
@@ -1534,7 +1534,8 @@ extern "C"
 
         SLANG_STRUCTURED_BUFFER             = 0x06,
         SLANG_BYTE_ADDRESS_BUFFER           = 0x07,
-        SLANG_RESOURCE_UNKNOWN              = 0x08,
+        SLANG_ACCELERATION_STRUCTURE        = 0x08,
+        SLANG_RESOURCE_UNKNOWN              = 0x09,
```

Comment:
While it makes some logical sense to have the `UNKNOWN` case come last, this edit changes the meaning of the `enum` cases between library versions. It would probably be safest to have `ACCELERATION_STRUCTURE` be added as `0x09`, and leave `UNKNOWN` as-is.